src/cromshell/status/command.py

```diff
@@ -32,19 +32,47 @@ def main(config, workflow_id):
     # Check if Cromwell Server Backend works
     http_utils.assert_can_communicate_with_server(config)
 
+    requested_status_json, workflow_status, updated_ret_val = query_status(
+        config, workflow_id
+    )
+    ret_val = updated_ret_val
+
+    # Display status to user:
+    line_string = requested_status_json
+    print(line_string.replace(",", ",\n"))
```

Comment:
Its fine as is but would be nice to use [pretty_print_json()](https://github.com/broadinstitute/cromshell/blob/ae1aa5c402335517779c529641c3b0c35b69e0e7/src/cromshell/utilities/io_utils.py#L113)